test/carotation/workloads/httpbin.yaml

```diff
@@ -63,7 +63,7 @@ spec:
     spec:
       serviceAccountName: httpbin
       containers:
-      - image: quay.io/joshvanl_jetstack/httpbin:latest
```

Comment:
comment: Yikes, good catch! I'd support moving this out of this PR and merging this immediately, but I'm also fine with including it here

Reply:
https://github.com/cert-manager/istio-csr/pull/226

Reply:
Will have to merge this PR first.

Reply:
Fair enough!